public/components/panel/studioPanel/beatUploader.js

```diff
@@ -0,0 +1,28 @@
+import { GridBeatObject, commitBeatIfNecessary } from '../workstationPanel/gridObjects'
+
+/**
+ * @param {String} beatTitle
+ * @param {File} beatImageFile
+ * @param {GridBeatObject} beatObject
+ * @param {(progress: Number) => void} onUploadProgress
+ * @param {(savedGridObject: GridBeatObject) => void} onUploadComplete
+ * @param {() => void} onError
+ */
+const BeatUploader = (beatTitle, beatImageFile, beatObject, onUploadProgress, onUploadComplete, onError) => {
+  // Temporary placeholder
+  // To be replaced with axios request to database which can
```

Comment:
Beats Uploader is unique to the studio workspace since that's the only place you can actually create a beat. It doesn't exist in the parts that @hungnguyen1895 and @gabeeee are working on. I agree with the shared package sentiment though. I have left other request handlers that are being worked on the homepage side of things and plan on just pulling on what's done by Hung &/ Gabe.